migrations/20160216120159_create_initial_schema.js

```diff
@@ -51,7 +51,7 @@ exports.up = (knex) => {
       .notNullable();
     table.text('study_design')
       .notNullable();
-    table.text('study_phase')
+    table.jsonb('study_phase')
```

Comment:
At a first glance at Knex documentation, we didn't see any mention about array types and observing
other tables in the database we opted to go with the jsonb route.
With your suggestion, we went back to the documentation and found the 'specificType' function that allows us to force the array type on
a column.
However, for an array of enum we would have to dive into using Raw SQL object as the enum function 
from Knex creates a constraint instead of an actual enum.
With this in mind, we think that the array of strings approach is better. We would like to know your thoughts on it.